### DISCUSSION TITLE
السلام عليكم

### DISCUSSION THREAD

**Discussion by MohammedRashwan97** [quran/discussions#12] [General]
![Screenshot_٢٠٢٤٠٥١٠_١٥١٦٤٣_Quran.jpg](https://github.com/quran/discussions/assets/158797731/893bf402-d243-41fa-aca5-7ed92c43c901)

التطبيق لما بخرج منه وانا فاتح صفحة تفسير وارجع افتحه من التطبيقات الأخيرة بيقفل صفحة التفسير وبحتاج أرجع أبحث على المكان ال كنت واقف عنده فى قراءة التفسير
فيه حاجة كمان فى صفحة المرجعيات فى مرجعيات الآيات ، الظاهر من الآية هما ٤ كلمات فقط وهذا خطأ كبير.  أنا أحفظ الكثير من الآيات التى أريد أن أعود إليها فيما بعد وانا لا أحفظ الآيات ولا أميزها فى ذهنى ، فعلشان أرجع لآية من اللى انا حافظهم فى المرجعيات ممكن افتح سبع آيات أو عشرة علشان أوصل للآية لأنى لا أستطيع تمييز الآية اللى عايز أوصل لها من أول أربع كلمات فيها. فأتمنى إن الآيات اللى محفوظة تكون ظاهرة كلها وليس أول ٤ كلمات فقط
![Screenshot_٢٠٢٤٠٥١٠_١٥٣٩١٤_Quran.jpg](https://github.com/quran/discussions/assets/158797731/6fbd3b40-94b7-4746-98f6-a8ff07e418b0)

**Comment by ahmedre** [comment]
وعليكم السلام ورحمة الله وبركاته
جزاكم الله خيراً
يمكننا تكملة الكلام هنا:
https://github.com/quran/quran_android/issues/2695

جزاكم الله خيراً
والسلام عليكم
